README.md

```diff
@@ -61,6 +61,25 @@ SolidusSquare::PaymentMethod.new(
   preference_source: "square_credentials"
 ).save
 ```
+### How to set the order updated webhook
+
+1. Visit the [SquareDeveloper](https://developer.squareup.com/apps) website.
+
+2. Login into your account.
+
+3. Create or open an existing app.
+
+4. Navigate on the left side navigation bar to `Webhooks`.
+
+5. Click on `Add Endpoiont`.
+
+6. Paste `<domain>/solidus_square/square_update` in the URL field, eg. `https://www.solidus.com/solidus_square/square_update`.
+
+7. Select/check `order.updated` from the Events.
+
+8. Click on `Save`.
+
+9. Navigate to the routes and uncomment the endpoint for the `order.updated` webhook.
```

Comment:
Considering we'll use the webhook only for the HostedCheckout, I think we should move these instructions under a specific section.

I think we can make a section `Hosted Checkout` and put these and the instruction to enable the checkout